code/src/index.css

```diff
@@ -1,13 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap');
+
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #BFFFF0;
+  font-family: "Quicksand", sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+ input[type='checkbox'] {
+	appearance: none;
+	-webkit-appearance: none;
+	position: relative;
+	padding: 10px;
+  margin-left: 10px;
+	border-radius: 50%;
+	height: 15px;
+	width: 15px;
+  border: 3px solid #000;
+  cursor: pointer;
 }
+
+input[type='checkbox']:checked {
+	background: #b0b0b0;
+  border: none;
+}
```

Comment:
Here it is! The secret to how you were making this satisfying "pop" when clicking on it - removing the border completely. Love it! I actually think this can be added into styled components. I think it would look something along these lines (not completely sure, would have to double check the documentation.)

```
const Input = styled.input.attrs({ 
  type: 'checkbox',
  value: 'checked'
})`
background: #b0b0b0;
border: none;`
```

The documentation can be found here I think: https://styled-components.com/docs/basics#attaching-additional-props